regression/pages/ecommerce/cybersource_page.py

```diff
@@ -0,0 +1,111 @@
+"""
+Cybersource page
+"""
+from bok_choy.page_object import PageObject
+
+from regression.pages.common.utils import (
+    fill_input_fields,
+    select_values_from_drop_downs,
+    click_checkbox
+)
+from regression.pages.ecommerce.back_to_basket_page import BackToBasketPage
+from regression.pages.ecommerce.cancel_checkout_page import CancelCheckoutPage
+
+
+class CyberSourcePage(PageObject):
+    """
+    Cybersource payment page
+    """
+
+    url = 'https://testsecureacceptance.cybersource.com/checkout'
```

Comment:
This should be in your constants file. For example `url = MY_CYBERSOURCE_CHECKOUT_URL`